next.config.js

```diff
@@ -94,7 +94,13 @@ module.exports = withSwingset({
       ],
     },
     images: {
-      domains: ['www.datocms-assets.com'],
+      formats: ['image/avif', 'image/webp'],
+      domains: [
+        'www.datocms-assets.com',
+        'mktg-content-api-hashicorp.vercel.app',
+      ],
+      dangerouslyAllowSVG: true,
+      contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
```

Comment:
This directive works specifically because everything is now going through the `/_next/image` path, right?